nav/stepper/motion.py

```diff
@@ -18,8 +17,35 @@ def __init__(self, GPIO):
 		[0.0.0] Rishav
 			--- Created function
 	"""
-	def mov(self, dir, distance):
-		print(dir, distance)
+	def mov(self, dir, distance, GPIO):
```

Comment:
Please remember to update Changelogs. For example: Why you added GPIO as an input variable.